integration-tests/splunk/README.adoc

```diff
@@ -0,0 +1,3 @@
+Splunk UI is helpful for testing.
```

Comment:
Can we expand on this a bit. Where do I get the UI from? Docker container? What's the command to get it running.

Reply:
I was probably a little bit plain about it. I tried to explain in the following sentence: `To access UI, expose port 8000 and access http://127.0.0.1:${exposed_8000)/ in a browser.` I agree that more information would be helpful

Reply:
fixed